WORKSPACE

```diff
@@ -84,6 +84,22 @@ maybe(
     url = "https://github.com/bazelbuild/rules_python/releases/download/0.1.0/rules_python-0.1.0.tar.gz",
 )
 
+maybe(
+    http_archive,
+    name = "rules_pkg",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
+        "https://github.com/bazelbuild/rules_pkg/releases/download/0.5.1/rules_pkg-0.5.1.tar.gz",
+    ],
+    sha256 = "a89e203d3cf264e564fcb96b6e06dd70bc0557356eb48400ce4b5d97c2c3720d",
+)
+
+load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
+rules_pkg_dependencies()
```

Comment:
Seems like this should go in dependency_support

Reply:
I tried. The problem I had is that I couldn't get `bazel test @rules_pkg//...` to work (at least locally) without registering an rpmbuild toolchain at the top level (which pushed the `rules_pkg` repo to the top level so you can load `rpmbuild_register_toolchains`. If you can think of a different way to deal with that, I'm all ears.